docs/community.md

```diff
@@ -36,6 +36,20 @@ Here are few ways, you can get involved.
  - Share new ideas/directions to pursue or propose a new HIP
  - Contributing code to the project ([newbie JIRAs](https://issues.apache.org/jira/issues/?jql=project+%3D+HUDI+AND+component+%3D+newbie))
 
+## Become a Committer
```

Comment:
Looks like we have to repeat this here, guess this does not evolve very frequently so it's ok to have it in 2 places and update..

Reply:
No.. this is the source.. html is generated off the md